### PR TITLE
1858 build charmhub button page

### PR DESCRIPTION
**Pull request by solazio**
## Done
- _Add publicise page with charmhub buttons._

## How to QA
- Run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://localhost:8045/[charm_name]/publicise

## Issue / Card
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1858

## Screenshots
image

### CODE REVIEW

File: static/js/src/publisher/publicise.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  dropDownSelector,
+  languageSectionListSelector
+) => {
+  const languagePicker = document.querySelector(dropDownSelector);
+  const languageSectionList = document.querySelectorAll(
+    languageSectionListSelector
+  );
+
+  const showLanguage = (language) => {
+    const open = document.querySelector("#" + language + "-content");
```
Comment (steverydz) [inline static/js/src/publisher/publicise.js:11]: I'd suggest using `#${language}-content`  here

**Review by steverydz — APPROVED** [review]
LGTM  :+1: 

Just one suggestion in the code

File: static/js/src/publisher/publicise.js (hunk 1 above)
Comment (solazio) [reply static/js/src/publisher/publicise.js]: agree

File: webapp/publisher/views.py [hunk 2]
```diff
@@ -13,6 +13,27 @@
 from webapp.decorators import login_required
 from webapp.helpers import get_licenses
 
+SUPPORTED_LANGUAGES = {
```
Comment (tbille) [inline webapp/publisher/views.py:16]: it doens't need to be a global variable. You can put it directly in the function for the endpoint

Comment (solazio) [reply webapp/publisher/views.py]: you are right

**Comment by spencerbygraves** [comment]
Thanks @solazio. Could we look at the space above the "Dark version/Light version" headers please? Currently it's a bit too close.

Here's the [design on Zeplin](https://zpl.io/2pwr3lQ).

Thanks.

**Pull request merged**